commands/Staff/wikiResync.js

```diff
@@ -0,0 +1,38 @@
+const { Command } = require('klasa')
+const { exec } = require('child_process')
+const rimraf = require('rimraf')
+
+module.exports = class extends Command {
+  constructor (...args) {
+    super(...args, {
+      name: 'wikiResync',
+      enabled: true,
+      runIn: ['text'],
+      cooldown: 30,
+      bucket: 1,
+      aliases: ['wikiupdate', 'wikipull', 'wikisync', 'updatewiki', 'syncwiki'],
+      permissionLevel: 0,
+      requiredPermissions: ['SEND_MESSAGES', 'CHANGE_NICKNAME'], // Lazy way to check for a staff role LMAO
```

Comment:
Don't think all staff have this perm anymore... Might be better to check agaist a list of role IDs?